src/test/java/dk/kb/datahandler/kaltura/KalturaManualFileUploader.java

```diff
@@ -0,0 +1,101 @@
+package dk.kb.datahandler.kaltura;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.apache.solr.client.solrj.SolrQuery;
+import org.apache.solr.client.solrj.SolrRequest.METHOD;
+import org.apache.solr.client.solrj.impl.Http2SolrClient;
+
+import org.apache.solr.client.solrj.response.QueryResponse;
+import org.apache.solr.common.SolrDocument;
+import org.apache.solr.common.SolrDocumentList;
+
+import com.kaltura.client.enums.MediaType;
+
+import dk.kb.kaltura.client.DsKalturaClient;
+
+public class KalturaManualFileUploader {
+
+    /**
+     * <p>
+     * Manual started job to upload the devel or stage collection of audio/video files to Kaltura. 
+     * The job takes a folder of audio or video files as input. 
+     * For each file title and description are fetched from Solr and used as meta-data in Kaltura
+     * The files will be uploaded with the metadata: tag=DS-KALTURA
+     * </p>
+     * 
+     *  This is a temporary solution for devel/stage to get data into Kaltura for the frontend.
+     *  
+     */
+    public static void main(String[] args)  {
+
+        String kalturaUrl= "https://kmc.kaltura.nordu.net";        
+        String adminSecret = "XXXXXXXXXXXXXXXXXXXXXXxxx"; 
+        Integer partnerId = 380; // Use this partner ID for DS project 
+        String userId = "XXX@kb.dk"; //User must exist in kaltura.                 
```

Comment:
Where are these values updated? Manually here? If so I would suggest defining them in the yaml configurations, so that configuration of the manual process is done there.

Reply:
Can not be defined. You do not know which kaltura environment you are uploading to. (test/stage/devel), also avoid having the kaltura mastersecret too many places.